user/papers/template.py

```diff
@@ -0,0 +1,63 @@
+pkgname = "papers"
+pkgver = "46.2"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dcomics=enabled",
```

Comment:
you can documentation=false and then don't need gi-docgen at all

Reply:
LOL no way